### PR TITLE
Reolink integration: Added device to docs

### PR DESCRIPTION
**Pull request by jorik12345**
## Proposed change
Added the Reolink E1 Outdoor Pro to the list of supported devices. I bought one myself and integrated it. I have tested all the functionalities and everything works as expected.

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
One thing I did noticed was that the device was not recognized through the integrations page.

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -171,6 +171,7 @@ The following models have been tested and confirmed to work:
 - C2 Pro
 - E1 Zoom
 - E1 Outdoor
+- E1 Outdoor Pro (Wi-Fi)
```
Comment (starkillerOG) [inline source/_integrations/reolink.markdown]: There is only one variant of the E1 Outdoor Pro, so no need to specify Wi-Fi, that is only used if there are 2 seperate variants.

```suggestion
- E1 Outdoor Pro
```

**Review by starkillerOG — DISMISSED** [review]
@jorik12345 thank you very much for reporting this model as working

**Review by starkillerOG — APPROVED** [review]
Can be merged

**Comment by starkillerOG** [comment]
If you appreciate the reolink integration and want to support its development, please consider [sponsering](https://github.com/sponsors/starkillerOG) the upstream library or purchase Reolink products through [this affiliate link](https://reolink.pxf.io/q44QWq).

File: source/_integrations/reolink.markdown (hunk 1 above)
Comment (jorik12345) [reply source/_integrations/reolink.markdown]: You are right, thanks for updating.

**Pull request merged**